.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
@krzysztof I added it to test based on @hachreak code https://github.com/hachreak/invenio-db/blob/json-support/.travis.yml.
When you look at what version travis starts it is still 5.6. So it seems that this like has no effect.